src/org/labkey/test/components/glassLibrary/navigation/FMBreadCrumbs.java

```diff
@@ -0,0 +1,210 @@
+package org.labkey.test.components.glassLibrary.navigation;
+
+import org.labkey.test.Locator;
+import org.labkey.test.components.Component;
+import org.labkey.test.components.WebDriverComponent;
+import org.labkey.test.components.react.DropdownButtonGroup;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * <p>
+ *     This is a navigational component for the breadcrumb.ts element <i>I think</i>.
+ * </p>
+ * <p>
+ *     The breadcrumb navigation component is an ordered list of elements <i>ol / li</i> where some of the elements
+ *     may be links and others are just text. It is also possible that some of the elements will have a dropdown menu
+ *     associated with them.
+ * </p>
+ */
+public class FMBreadCrumbs extends WebDriverComponent<FMBreadCrumbs.ElementCache>
+{
+    final WebElement _breadCrumbsElement;
+    final WebDriver _driver;
+
+    public FMBreadCrumbs(WebDriver driver)
+    {
+        this(Locators.component.findElement(driver), driver);
+    }
+
+    private FMBreadCrumbs(WebElement element, WebDriver driver)
+    {
+        _breadCrumbsElement = element;
+        _driver = driver;
+    }
+
+    @Override
+    protected WebDriver getDriver()
+    {
+        return _driver;
+    }
+
+    @Override
+    public WebElement getComponentElement()
+    {
+        return _breadCrumbsElement;
+    }
+
+    /**
+     * Private function that returns all of the li/span elements in this ol element.
+     *
+     * @return List of span tags in the ordered list.
+     */
+    private List<WebElement> getBreadCrumbElements()
+    {
+        return Locators.breadCrumb.findElements(this);
+    }
+
+    /**
+     * Return the text value of the breadcrumbs.
+     *
+     * @return List containing the text of the breadcrumbs.
+     */
+    public List<String> getBreadCrumbs()
+    {
+        List<String> breadCrumbs = new ArrayList<>();
+        for(WebElement crumb : getBreadCrumbElements())
+        {
+            breadCrumbs.add(crumb.getText());
+        }
+
+        return breadCrumbs;
```

Comment:
We have a helper for this sort of thing:
```suggestion
        return getWrapper().getTexts(getBreadCrumbElements());
```

Reply:
Changed.